package.json

```diff
@@ -105,13 +110,23 @@
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "has-bigints": "^1.0.2",
     "husky": "^7.0.2",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^27.4.3",
     "jest-axe": "^6.0.0",
+    "jest-resolve": "^27.4.2",
+    "jest-watch-typeahead": "^1.0.0",
     "lint-staged": "^13.0.3",
+    "postcss": "^8.4.24",
+    "postcss-flexbugs-fixes": "^5.0.2",
+    "postcss-normalize": "^10.0.1",
+    "postcss-preset-env": "^7.8.2",
     "prettier": "^2.8.8",
-    "react-app-rewired": "^2.2.1",
+    "react-app-polyfill": "^3.0.0",
```

Comment:
hmmm.... what is react-app-polyfill doing for us?  I thought we are fully ejecting from CRA...

Reply:
It was added by CRA's eject. It polyfills `fetch` in JSDOM.

We're no longer using `react-scripts`, but some of the other CRA packages are still used after ejecting (`react-app-polyfill`, `babel-preset-react-app`, etc).

Reply:
Easy enough to get rid of though. https://github.com/DataBiosphere/terra-ui/pull/3989/commits/85a188a55cf2d2e3357fe29b593990c2f15e5bef.

Reply:
Assuming that `whatwg-fetch` is preferable… last publish was 2 years ago.

Reply:
All `react-app-polyfill/jsdom` does is require `whatwg-fetch`.
https://github.com/facebook/create-react-app/blob/main/packages/react-app-polyfill/jsdom.js

Reply:
Ah, well in that case… Thanks, Nick.